lib/chromium/root.js

```diff
@@ -180,10 +180,12 @@ var ChromiumTabActor = ActorClass({
   },
 
   form: function(detail) {
+    // Change chrome-specific URLs to something that Firefox can handle.
+    let url = this.json.url.replace(/^chrome:\/\//, "http://chrome-");
```

Comment:
hum that's weird to replace it with non-existing URL but I don't have any better idea.
It's ok as I think url is only used for displaying some labels.